src/mongocxx/test/spec/client_side_encryption.cpp

```diff
@@ -61,6 +61,11 @@ void add_auto_encryption_opts(document::view test, options::client* client_opts)
                 test_encrypt_opts["bypassAutoEncryption"].get_bool().value);
         }
 
+        if (test_encrypt_opts["bypassQueryAnalysis"]) {
+            auto_encrypt_opts.bypass_query_analysis(
+                test_encrypt_opts["bypassQueryAnalysis"].get_bool().value);
+        }
```

Comment:
```suggestion
        if (const auto bqa = test_encrypt_opts["bypassQueryAnalysis"]) {
            auto_encrypt_opts.bypass_query_analysis(bqa.get_bool().value);
        }
```
Avoid double-lookup.